crt_portal/cts_forms/templates/forms/complaint_view/show/actions/comment_summary.html

```diff
@@ -0,0 +1,27 @@
+<div class="crt-portal-card">
+  <div class="crt-portal-card__content">
+    <form
+      id="comment-actions-{{ id_name }}"
+      class="usa-form"
+      method="post"
+      action="/form/comment/report/{{ data.pk }}/"
```

Comment:
Opportunity to avoid the hard coded url here and build this url w/ a template tag.
Something like...
``` 
{% url 'save-report-comment' report_id=data.pk %}
```